### PR TITLE
Improve web client

### PR DESCRIPTION
**Pull request by seqre**
* css moved to style.css
* aligned sizes of entities to match each other
* reformatted html
* added Thymeleaf

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -22,13 +14,12 @@
...
 
 <section>
-
 </section>
 
 <script>
     // For localhost testing
-    // const proxyURL = "https://cors-anywhere.herokuapp.com/";
-    const proxyURL = "";
+    const proxyURL = "https://cors-anywhere.herokuapp.com/";
```
Comment (hjaremko) [inline src/main/resources/templates/index.html:21]: Revert

File: src/main/resources/templates/index.html [hunk 2]
```diff
@@ -79,15 +70,17 @@
...
 <h1> Sweet Animals </h1>
-<label for="output"></label><textarea id="output" rows="24" cols="80"></textarea>
-<br>
-<label for="input"></label><input type="text" id="input" size="80"/>
-<button id="sendButton" onclick="sendRequest()">Send</button>
 
-<link rel="stylesheet" href="https://maxbeier.github.io/text-spinners/spinners.css">
+<div>
+    <label for="output"></label><textarea cols="80" id="output" rows="24"></textarea>
+    <label for="input"></label><input id="input" size="74" type="text"/>
```
Comment (hjaremko) [inline src/main/resources/templates/index.html:81]: This fixes nothing, these sizes should be based on percentages or something. To do in the future.

Comment (seqre) [reply src/main/resources/templates/index.html:81]: This whole website is one big TODO in the future. And you're right, it doesn't fix anything, I've just fix size of input and button to fit to the output width.

Comment (hjaremko) [reply src/main/resources/templates/index.html:81]: It was aligned perfectly on my machine 🤣 I believe it depends on the font

**Pull request merged**